build.xml

```diff
@@ -123,6 +122,19 @@
     </uptodate>
   </target>
 
+  <path id="junit.build.classpath">
+    <fileset dir="${lib.dir}">
+      <include name="junit-4.11.jar"/>
+    </fileset>
+  </path>
+
+  <path id="junit.run.classpath">
+    <path refid="junit.build.classpath"/>
+    <fileset dir="${lib.dir}">
+      <include name="hamcrest-core-1.3.jar"/>
```

Comment:
We use no custom matchers in our tests, although JUnit uses it under the covers. I suppose I could add it to the compile path in case a connector's unit tests want to use custom matchers. Another possibility compilation succeeded might be that it was picking up the ham crest jar from Corbatura.